src/main/java/com/puppycrawl/tools/checkstyle/XMLLogger.java

```diff
@@ -208,7 +208,7 @@ private void writeFileError(AuditEvent event) {
                 + event.getSeverityLevel().getName()
                 + "\"");
         writer.print(" message=\""
-                + encode(event.getMessage())
+                + event.getMessage()
```

Comment:
Find inspiration at this update https://github.com/checkstyle/checkstyle/commit/d11b600d2aad3a8ba1ef76a252024758bbdc5edc